src/helpers.coffee

```diff
@@ -103,46 +112,26 @@ exports.normalizeKeystrokes = (keystrokes) ->
   normalizedKeystrokes.join(' ')
 
 exports.keystrokeForKeyboardEvent = (event, dvorakQwertyWorkaroundEnabled) ->
-  unless KeyboardEventModifiers.has(event.keyIdentifier)
-    charCode = charCodeFromKeyIdentifier(event.keyIdentifier)
-
-    if dvorakQwertyWorkaroundEnabled and typeof charCode is 'number'
-      charCode = event.keyCode
-
-    if charCode?
-      if process.platform is 'linux' or process.platform is 'win32'
-        charCode = translateCharCodeForWindowsAndLinuxChromiumBug(charCode, event.shiftKey)
-
-      if event.location is KeyboardEvent.DOM_KEY_LOCATION_NUMPAD
-        # This is a numpad number
-        charCode = numpadToASCII(charCode)
-
-      charCode = event.which if not isASCII(charCode) and isASCII(event.keyCode)
-      key = keyFromCharCode(charCode)
-    else
-      key = event.keyIdentifier.toLowerCase()
+  prep_event = new PreprocessableKeyboardEvent event
```

Comment:
I'd prefer to call a method that returns an object rather than building an object that we then mutate. Also, variables in our codebase are `camelCased`.